src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -46,7 +46,8 @@
  * <p>
  * may optionally be exempted from the policy using the {@code allowEmptyMethods},
  * {@code allowEmptyConstructors}, {@code allowEmptyTypes}, {@code allowEmptyLoops},
- * {@code allowEmptyLambdas} and {@code allowEmptyCatches} properties.
+ * {@code allowEmptyLambdas}, {@code allowEmptyCatches}
+ * and {@code allowEmptySwitchCase} properties.
```

Comment:
The JLS should be our source of truth when referring to language elements. This property should be called `allowEmptySwitchBlock`. Please see https://docs.oracle.com/javase/specs/jls/se21/html/jls-14.html#jls-14.11.1

Reply:
> The body of both a switch statement and a switch expression ([§15.28](https://docs.oracle.com/javase/specs/jls/se21/html/jls-15.html#jls-15.28)) is called a switch block.

I think this name means `switch(x) {}` not 
```
switch(x) {
  case 1: {}
}
```
`allowEmptySwitchLabel` ?

Reply:
The token that represents `{}` is called either a block (for -> switch) or block statement (for traditional : ).